client/client.js

```diff
@@ -59,8 +59,10 @@ function CloudCmdProto(DOM) {
     this.prefixSocket = '';
     this.prefixURL = '';
     
-    this.MIN_ONE_PANEL_WIDTH = 1155;
-    this.MOBILE_ONE_PANEL_WIDTH = 600;
+    var bodyStyle = getComputedStyle(document.body);
+    this.MIN_ONE_PANEL_WIDTH = bodyStyle.getPropertyValue("--min-one-panel-width");
+    this.MOBILE_ONE_PANEL_WIDTH = bodyStyle.getPropertyValue('--mobile-max-width')
+    
```

Comment:
Not sure i understand your comment.

I think this is ready for merge as it stands.

Improving CSS "code" (to reduce 600px repetition) can be done on other PR as i mentioned on https://github.com/coderaiser/cloudcmd/issues/423

Reply:
I mean that `innerWith` check can be avoided, since css already does all checks, and it can set css, and from js we can just read boolean value when needed.

Reply:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units

We can use 0 and 1 as true and false